converters/generic_farm_calendar_aim.jsonld

```diff
@@ -0,0 +1,427 @@
+{
+    "@context": [
+        "https://w3id.org/ocsm/main-context.jsonld"
+    ],
+    "@graph": [
+        {
+            "@id": "urn:openagri:farm:563493ef-cf51-41d9-8775-3b80a8394a07",
+            "@type": "Farm",
+            "name": "The OpenAgri prototype Farm",
+            "description": "A farm in Attica Greece with two parcels cultivating table olives",
+            "hasAdministrator": "Marinos Antypas",
+            "contactPerson": {
+                "@id": "urn:openagri:contact:36628afe-64e0-45d3-899b-084975e806c0",
+                "@type": "Person",
+                "firstname": "Marinos Antypas",
+                "lastname": "Marinos Antypas"
+            },
+            "telephone": "+30-08101907",
+            "vatID": "xyz-abc-1234",
+            "address": {
+                "@id": "urn:openagri:address:fe3f4fdd-65f8-4aed-a527-e66de5af098c",
+                "@type": "Address",
+                "adminUnitL1": "Greece",
+                "adminUnitL2": "Attica",
+                "addressArea": "no address",
+                "municipality": "Oropos",
+                "community": "",
+                "locatorName": "Killeler"
+            },
+            "area": 200000,
+            "hasAgriParcel": [
+                {
+                    "@id": "urn:openagri:parcel:32989475-9b30-494e-b82b-01360f924803",
+                    "@type": "Vineyard",
+                    "identifier": 123001,
+                    "description": "Olive grove A",
+                    "category": "orchard",
+                    "validFrom": "2023-01-01T00:00:00",
+                    "validTo": "2023-12-31T23:59:59",
+                    "inRegion": "Attica/Greece",
+                    "hasToponym": "Oropos",
+                    "area": 30000.0,
+                    "isNitroAarea": 1,
```

Comment:
Wouldn't it make sense to have this as: "isNitroAParcel" or simply "isNitroA"? since we are going with the "Parcel" instead of area? this also help avoiding confusion with the "area" property from both the Farm and the Parcel.

Reply:
I believe that this is more related to the ontology rather than the conversion script, so probably something for @rapw3k

Reply:
or maybe @nikoskal can contribute on that